authors/urls.py

```diff
@@ -20,5 +20,6 @@
     path('admin/', admin.site.urls),
 
     path('api/', include(('authors.apps.authentication.urls', 'authentication'), namespace='authentication')),
+    path('api/', include(('authors.apps.profiles.urls', 'authentication'), namespace='profiles')),
```

Comment:
Check the second parameter passed in the `include`  tuple. From [Django's documentation](https://docs.djangoproject.com/en/2.1/ref/urls/#include), it ought to be:
```python
 include((pattern_list, app_namespace), namespace=None)
```
Just change the `authentication` to `profiles` and we'll be okay on this.